src/Client/Web/Bit.Client.Web.BlazorUI/Components/Labels/BitLabel.razor.cs

```diff
@@ -6,8 +6,14 @@ public partial class BitLabel
     {
         private bool isRequired;
 
+        /// <summary>
+        /// A label for a form element
+        /// </summary>
         [Parameter] public string? For { get; set; }
 
+        /// <summary>
+        /// Whether the associated form field is required or not
```

Comment:
this description is not correct. we need to describe what this parameter actually does.